functions/cloud_functions/user.js

```diff
@@ -56,4 +56,24 @@ exports.userCreated = functions.auth.user().onCreate(async user => {
         return console.log(err)
     }
 
-})
\ No newline at end of file
+})
+
+exports.userApproved = functions.firestore.document("/users/{userId}/private/info").onUpdate(async (change, context) => {
+    try {
+        await admin.firestore().doc(`newUser/${change.before.data().email}`).delete();
+    } catch (error) {
+        console.log(error.message)
+    }
+});
```

Comment:
we don't need this cloud function...pay no attention to `newUser` collection, it's just a dumping group.

Besides, anytime an admin updates a user, even though he wasn't approving a user, this function will run...so it's a waste of read and write activities.